# Comprehensive Molecular Characterization of Perineural Invasion in Cervical Cancer

This repository contains the data and code used for the analysis presented in the paper: Comprehensive Molecular Characterization of Perineural Invasion in Cervical Cancer.

The raw sequence data reported in this paper have been deposited in the Genome Sequence Archive (Genomics, Proteomics & Bioinformatics 2021) in National Genomics Data Center (Nucleic Acids Res 2022), China National Center for Bioinformation / Beijing Institute of Genomics, Chinese Academy of Sciences (GSA-Human: HRA010315). Please access it from the following link:
https://bigd.big.ac.cn/gsa-human/browse/HRA010315


### The environment and software version details 

- miniconda3 4.10.1
  - Python 3.8.3
  - R 3.6.3
  
- RNAseq Analysis
  - fastp 0.20.1
  - TopHat2 2.1.1
  - Picard 2.23.8
  - HTSeq 0.11.3
  - Cufflinks 2.2.1
  - DESeq2
  
- WES Analysis
  - fastp 0.20.1
  - bwa 0.7.17
  - GATK 3.8.1
  - MC3 Ellrott K, Bailey MH, Saksena G, Covington KR, Kandoth C, Stewart C, et al. Scalable Open Science Approach for Mutation Calling of Tumor Exomes Using Multiple Genomic Pipelines. Cell Syst 2018;6:271-81 e7
  - deconstructSigs 1.8.0
  - CNVkit 0.9.7
  - GISTIC2 2.0.23
  - pctGCdata 0.2.0
  - FACETS 0.5.14

  
- WGS Analysis
  - fastp 0.20.1
  - bwa-0.7.17
  - GATK 3.8.1
  - CNVkit 0.9.7
  - GISTIC2 2.0.23
  
  
  
  
  
  
